### PR TITLE
F make receive station html

### PR DESCRIPTION
**Pull request by denysgerasymuk799**
Could you check my styles and colors. Write some comments to fix them

### CODE REVIEW

**Review by elenalazarus — CHANGES_REQUESTED** [review]
1) Very uncomfortable position of the window. Place it in the middle until the profile of the organisation is fully created
2) Categories field does not coincide with one at the mockup

File: web_server/receive_form.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+                    and request.form.get("time_out") and request.form.get("description"):
+                input_data = {'user_id': 123456,
+                              'type_id': 123456,
+                              'categories': request.form.getlist("categories"),
+                              'locations': [request.form.get('city'), request.form.get('address')],
+                              'time_from': request.form.get('time_to'),
+                              'time_to': request.form.get('time_out'),
+                              'description': request.form.get('description')}
+                print(input_data)
+                return input_data
```
Comment (alazaniia) [inline web_server/receive_form.py]: Don't return it yet.

File: web_server/static/style_for_receive_station.css [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    flex-basis: 100%;
+    flex: 1
+}
+
+.double-column {
+    display: flex;
+    flex-direction: column;
+    flex-basis: 100%;
+    flex: 2
+}
```
Comment (alazaniia) [inline web_server/static/style_for_receive_station.css:114]:
```
ul.chosen-choices {
    border-radius: 10px;
    min-height: 40px;
    display: flex;
    align-items: center;
}

li.search-choice > span {
    font-family: inherit;
    font-weight: normal;
    font-size: 110%;
}


ul.chosen-choices > li.search.choice {
    border-radius: 10px;
}


ul.chosen-choices > li {
    display: flex;
    align-items: center;
}

select {
    background-color: white;
}

div.checkboxes {
    text-align: left;
}

div.chosen-container, div.chosen-container-multi {
    margin: 8px 0;
}
```

File: web_server/static/style_for_receive_station.css [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+    border: 1px var(--light-powder);
+    border-radius: 15px;
+    box-sizing: border-box;
+}
+
+div.calender{
+    width: 100%;
+    margin: 0px 0px 30px 0px;
+}
+
```
Comment (alazaniia) [inline web_server/static/style_for_receive_station.css:60]:
```
div.calender > input {
    border-radius: 10px;
    border-color: transparent;
}
```

File: web_server/static/style_for_receive_station.css [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+input[type=text], select {
+    width: 100%;
+    height: 40px;
+    padding: 12px 20px;
+    font-size: 110%;
+    margin: 8px 0;
+    display: inline-block;
+    border: 1px var(--light-powder);
```
Comment (alazaniia) [inline web_server/static/style_for_receive_station.css:40]:
```
    display: flex;
    border: solid 1px var(--grey);
```

File: web_server/static/style_for_receive_station.css [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+}
+
+label[for=address], label[for=type-org]{
+    text-align: right;
+}
+
+input[type=text], select {
+    width: 100%;
+    height: 40px;
+    padding: 12px 20px;
```
Comment (alazaniia) [inline web_server/static/style_for_receive_station.css]:
```
    padding-left: 12px;
```

**Review by alazaniia — CHANGES_REQUESTED** [review]

File: web_server/static/style_for_receive_station.css [hunk 6]
```diff
@@ -53,9 +53,9 @@ textarea[type=text-description] {
     box-sizing: border-box;
 }
 
-div.calender{
-    width: 100%;
-    margin: 0px 0px 30px 0px;
+div.calender > input {
+    border-radius: 10px;
+    border-color: transparent;
 }
```
Comment (alazaniia) [inline web_server/static/style_for_receive_station.css:60]: please add this to align the Categories* label
```
div.checkboxes {
    text-align: left;
}
```

**Review by alazaniia — APPROVED** [review]
good job

**Review by sofiia-tesliuk — CHANGES_REQUESTED** [review]
Please, finish fixes, commented above

**Review by sofiia-tesliuk — APPROVED** [review]

**Review by LAnastasiia — APPROVED** [review]

**Pull request merged**